packages/shopping-cart/src/types.ts

```diff
@@ -867,6 +867,8 @@ export interface ResponseCartProductExtra {
 	legal_agreements?: never[] | DomainLegalAgreements;
 	is_gravatar_domain?: boolean;
 	is_hundred_year_domain?: boolean;
+	// which domain vendor (Donuts, Verisign, etc.) generated this domain suggestion
+	domain_vendor?: string;
```

Comment:
In domain suggestions we have `vendor` and `root_vendor`. What we want to log for the purposes of this project is the `root_vendor`. Is that what we are logging here? If so, would it make sense to call it `root_vendor` so we won't confuse it with `vendor`?